soc/arm/st_stm32/stm32f4/dts_fixup.h

```diff
@@ -523,4 +523,12 @@
 #define DT_CAN_1_CLOCK_BUS		DT_ST_STM32_CAN_40006400_CLOCK_BUS
 #define DT_CAN_1_CLOCK_BITS		DT_ST_STM32_CAN_40006400_CLOCK_BITS
 
+#define DT_CRYP_0_NAME			DT_INST_0_ST_STM32_CRYP_LABEL
```

Comment:
Can the driver not use `DT_INST_` defines directly and not require any dts_fixup.h changes?

Reply:
Removed dts_fixup.h changes.